src/components/Inputs/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import BaseInput from '../BaseInput';
+
+interface TextInputProps {
+  size?: 'large' | 'small';
+  placeholder?: string;
+  state?: 'idle' | 'disabled' | 'error' | 'success';
+  value: string;
+  // eslint-disable-next-line no-unused-vars
```

Comment:
I wouldn't say I like these comments, I prefer to disable this from the config file if needed.

Reply:
There isn't a problem with this rule to turn it off in general, it's just not needed in this particular case.